test/model/message_test.dart

```diff
@@ -162,6 +162,34 @@ void main() {
   });
 
   group('handleUpdateMessageEvent', () {
+    test('update timestamps on all messages', () async {
+      const t1 = 1718748879;
+      const t2 = t1 + 60;
+      final message1 = eg.streamMessage(lastEditTimestamp: null);
+      final message2 = eg.streamMessage(lastEditTimestamp: t1);
+      // This event is a bit artificial, but convenient.
+      // TODO use a realistic move-messages event here
+      final updateEvent = Event.fromJson({
+        ...eg.updateMessageEditEvent(message1).toJson(),
```

Comment:
@PIG208 as part of #171 you'll be writing a helper for conveniently making `UpdateMessageEvent` values representing a move (an `eg.updateMessageMoveEvent` similar to the existing `eg.updateMessageEditEvent`). So then you can follow that with a small commit fixing this TODO.